### PR TITLE
Update README.md with testing instructions

### PR DESCRIPTION
**Pull request by jpalawaga**
This change is [Reviewable](https://reviewable.io/reviews/plangrid/plangrid-api-net/12)

### CODE REVIEW

**Comment by jpalawaga** [comment]
I wasn't sure whether or not if we wanted to put the environment variables directly into the readme given the public nature of their contents.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,21 @@ To get the next 20 projects (after the first 50), you would call:
...
+
+You're free to open a pull-request and have the rest runner run the suite of tests
+for you. If you want to run the tests locally, you will need to setup a few things.
+
+1. NUnit 2 Test Adapter. You can install the test adapter in your IDE by
+opening the Extensions manager (Tools > Extensions and Updates).
+
+1. Visual Studio needs to be configured to run as an Administrator. You can follow
+the directions on this [StackOverflow](https://stackoverflow.com/questions/12257110)
+question is you need assistance.
```
Comment (ViktoriyaSavkina) [inline README.md]: FYI I didn't have to run VS as an administrator

Comment (kirkplangrid) [reply README.md:70]: Yeah, why would that be required?  Curious.

Comment (jpalawaga) [reply README.md:70]: @kirkplangrid my test suite was not discovering the tests, remember? I had to run as administrator to get it to find em

**Pull request closed without merging**